Tasks/PackerBuildV1/package-lock.json

```diff
@@ -0,0 +1,311 @@
+{
```

Comment:
hopefully you have just copied the lock file rather than generating it again - doing so can  pick a later version of modules leading to OSS governance issues.

Reply:
reverted to package-lock from Task version V0